walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -45,89 +46,96 @@
       }
     }
 
-    Collections.sort(conflicts, TimeRange.ORDER_BY_START);
-
-    return conflicts;
+    return ImmutableList.sortedCopyOf(
+        TimeRange.ORDER_BY_START.thenComparing(TimeRange.ORDER_BY_END.reversed()), conflicts);
   }
 
-  public List<TimeRange> mergeOverlappingRanges(List<TimeRange> conflicts) {
+  private ImmutableList<TimeRange> mergeOverlappingRanges(List<TimeRange> conflicts) {
```

Comment:
Add a comment to this method that you require the conflicts to be sorted by start time then end time. Otherwise it's not obvious to the reader.

Or, do the sorting in this method.